src/App.tsx

```diff
@@ -1,223 +1,101 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Footer } from './components/Auth/todos/Footer';
+import { Header } from './components/Auth/todos/Header';
+import { TodoList } from './components/Auth/todos/TodoList';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
-  const newTodoField = useRef<HTMLInputElement>(null);
 
-  useEffect(() => {
-    // focus the element with `ref={newTodoField}`
-    if (newTodoField.current) {
-      newTodoField.current.focus();
+  const [addError, setAddError] = useState(true);
+  const [deleteError, setDeleteError] = useState(false);
+  const [updateError, setUpdateError] = useState(false);
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState('All');
```

Comment:
Do not rely on the unknown string, make constants for this. Better crate enum for this purpose and use it all over the code when you work with `status` or filter types. When you would write test tasks, using `enum` in similar case would be a ➕  and could help you to get a job offer 😉